app/views/content_items/guide.html.erb

```diff
@@ -4,6 +4,23 @@
     canonical_url: @content_item.canonical_url,
     body: @content_item.has_parts? ? @content_item.current_part_body : nil
   ) %>
+
+  <%
+    scroll_track_headings_paths = [
+      "/settled-status-eu-citizens-families",
+      "/council-housing",
+      "/family-permit",
+      "/driving-abroad",
+      "/get-a-passport-urgently",
+      "/driving-abroad/international-driving-permit",
+      "/taking-goods-out-uk-temporarily/get-an-ata-carnet",
+    ]
+
+    full_url = [@content_item.base_path, @content_item.part_slug].join('/')
```

Comment:
I'm not sure this will operate correctly, given the paths you have listed above. 

I assume we're expecting part_slug to sometimes be nil in order to have paths like "/council-housing", however if we do `["/council-housing", nil].join("/")` we get `/council-housing/`. Perhaps you want to do `[@content_item.base_path, @content_item.part_slug].compact.join('/')` ?

Reply:
Great spot, thanks! Have updated.

Reply:
The trello card links to a spreadsheet of all these URLs and some more details about them, including what the original tracking was and what changes have been made. Each one falls into one of these categories:

- not needed and scroll tracking has (probably already) been removed
- was tracking headings but content has changed so no tracking was occurring, so have removed tracking
- was tracking headings or percentages so has been migrated to the new scroll tracker